packages/auth/src/graphql/schema.graphql

```diff
@@ -40,6 +43,14 @@ input PaginationInput {
   last: Int
 }
 
+input GrantFilter {
+  identifier: FilterString
```

Comment:
@sabineschaller what do you think about always requiring `identifier` in any Open Payments Grant Request? Does it make sense to allow issuing grants for any payment pointer?